sfyra/README.md

```diff
@@ -4,16 +4,15 @@ Integration test for Sidero/Arges.
 
 ## Running
 
-Build the test binary:
+Build the test binary and Sidero, push images:
 
-    make sfyra
+    make USERNAME=<username> PUSH=true
```

Comment:
Is the function of the USERNAME environment variable explained somewhere in the rest of the documentation?

Reply:
it's kind of convention across all our repos, image is `$(REGISTRY)/$(USERNAME)/<imagename>`